ui/src/components/credentials/CredentialDetails.tsx

```diff
@@ -216,6 +228,11 @@ export function CredentialDetails() {
                       text={expiresAt ? formatDate(expiresAt) : "-"}
                     />
 
+                    <Detail
+                      label="Refresh Service"
+                      text={refreshService ? refreshService.id : "-"}
```

Comment:
Aren't we missing the `href` here?

I also think you're missing it in `ImportSchemaPreview.tsx`, `ImportSchemaPreview.tsx` and `SchemaDetails.tsx`.